extensions/amp-fresh/0.1/amp-fresh-manager.js

```diff
@@ -14,14 +14,99 @@
  * limitations under the License.
  */
 
+import {addParamToUrl} from '../../../src/url';
 import {fromClass} from '../../../src/service';
+import {isExperimentOn} from '../../../src/experiments';
+import {whenDocumentReady} from '../../../src/document-ready';
+import {user} from '../../../src/log';
+import {xhrFor} from '../../../src/xhr';
 
 
+/** @const */
+const TAG = 'amp-fresh';
+
 /**
  * Manages registered amp-fresh components and does the xhr request
  * and update.
  */
-export class AmpFreshManager {}
+export class AmpFreshManager {
+
+  /**
+   * @param {!Window} win
```

Comment:
where would i call ampdocFor(window) here on the constructor or on the install function?